argo-data-submission-wf/main.nf

```diff
@@ -131,6 +135,12 @@ upload_params = [
   'first_retry_wait_time': params.first_retry_wait_time,
   'cpus': params.cpus,
   'mem': params.mem,
```

Comment:
definitions of `cpu` and `mem` are not needed as upload subwf does not contain these two params.